README.md

```diff
@@ -167,6 +167,13 @@ Prometheus ideas:
 -	Mentor(s): Goutham Veeramachaneni (@gouthamve) / Krasi Georgiev (@krasi-georgiev)
 -	Issue: https://github.com/prometheus/tsdb/issues/234
 
+#### Allow index files larger than 64GB.
+
+-	This would require some code refactoring on how the index file is saved to disk using varints or some compression.
+-	Recommended Skills: Golang, Compression algorithms
```

Comment:
Compression isn't relevant here, no matter what compression we have there'll always be a limit. We should have no limit.

Reply:
updated